LifeTimeExample.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.24;
+
+contract LifeTimeExample {
+    address public owner;
+    uint storedData;
+    bool public freeze;
+
+    constructor() public {
+        owner = msg.sender;
+        freeze = false;
+    }
+
+    modifier CheckFreeze {
+        require(freeze == false, "This function is freezed.");
+        _;
+    }
+
+    function setFreeze() public {
```

Comment:
Just `Freeze` and `Unfreeze`

Reply:
OK

any more advices?

Reply:
That's all.

Reply:
`fn freeze()` have the name with `bool public freeze`, so i design setXxx

Reply:
freeze should be `freezed`